@xen-orchestra/fs/src/local.js

```diff
@@ -92,7 +92,10 @@ export default class LocalHandler extends RemoteHandlerAbstract {
   }
 
   async _readFile(file, options) {
-    return fs.readFile(this._getFilePath(file), options)
+    return retry(() => fs.readFile(this._getFilePath(file), options), {
+      tries: 2,
```

Comment:
Why `2`? What does `graceful-fs` does?

Reply:
I had chosen `2 tries` because for me, if the read are failing 2 time, it's probably because he have a real issue with the file he try to read. Maybe am wrong, but i didn't found the behaviour of graceful-fs.

Reply:
https://github.com/isaacs/node-graceful-fs/blob/24f88fd7f37829d34ba5df5edfd7fe20df55d74a/polyfills.js#L128

Reply:
Thank's.

Should we implement the same behaviour by default ?

Reply:
You already did it but I'm answering for the future :slightly_smiling_face: 

We don't need to do the same thing, but it's a good place to start.